### ISSUE TITLE
i want to know why the lr for SGD or MSGD is 0.01,rather than 0.1

### ISSUE THREAD

**Issue by solon37** [fsschneider/DeepOBS#37]
The paper provided not 0.1? or 0.01 has better performance in 100 epochs.
![image](https://user-images.githubusercontent.com/60464034/122538876-95deb780-d059-11eb-86f3-317c868cb5b4.png)

**Comment by fsschneider** [comment]
Hi,

I am not sure, I fully understand your question.
The `oneshot` budget uses default values, so no tuning is done. Instead, we use "default" values, that many optimizers now explicitly come with (e.g. Adam has clearly defined default hyperparameters).
In the case of `SGD` it is a little less clear. But a learning rate of `0.01` has been considered "default", e.g. it is used as the default learning rate in the TensorFlow implementation (see [the documentation](https://www.tensorflow.org/api_docs/python/tf/keras/optimizers/SGD))

**Comment by solon37** [comment]
Thank you!
You are right, the default learning rate in TensorFlow is 0.01, but Pytorch is 0.1. When i change the learing rate to 0.1,it may be better.
